src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+@import "../variables/variables";
+
+.calendar {
+  margin: 100px auto;
```

Comment:
Calendar should be centered - in the middle of the screen. Maybe try to add flex to body?